<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	
	
	background-image: url(:/new/icon/images/26.jpg);
}</string>
  </property>
  <widget class="QLineEdit" name="UsernameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>144</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="NicknameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NicknameLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>144</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>144</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordCheckLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码验证</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordCheckLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>144</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="UsernameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>105</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请填写注册信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius:10px;
font: 9pt &quot;Consolas&quot;;
	background-color: rgb(170, 170, 255);
}
QPushButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,225,255,200);
color:rgba(0,0,0,200);
}
QPushButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,225,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-radius:10px;
font: 9pt &quot;Consolas&quot;;
	background-color: rgb(170, 170, 255);
}
QPushButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,225,255,200);
color:rgba(0,0,0,200);
}
QPushButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,225,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
